src/migrationConvertLogic.ts

```
@@ -0,0 +1,106 @@
+        // created_at と updated_at の場合で now() がデフォルト値の時は DB::raw を使用
+        if (
+          (column.name === "created_at" || column.name === "updated_at") &&
+          column.default.toLowerCase() === "now()"
+        ) {
+          def += `->default(\\DB::raw('now()'))`;
+        } else if (column.type === "integer" || column.type === "int") {
+          // 整数型の場合、数値として扱う
+          def += `->default(${parseInt(column.default)})`;
+        } else if (column.type === "boolean") {
+          // 真偽値の場合、booleanとして扱う
+          def += `->default(${column.default.toLowerCase() === "true"})`;
```

Biome (1.9.4)

[error] Use Number.parseInt instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.parseInt instead.


(lint/style/useNumberNamespace)